config/initializers/assets.rb

```diff
@@ -12,3 +12,5 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( users/profile.js )
+Rails.application.config.assets.precompile += %w( users/profile_edit.js )
```

Comment:
Can we simplify this to one line: `Rails.application.config.assets.precompile += %w( users/profile.js users/profile_edit.js )`?

Reply:
yeah, I'll take care of that.